src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
```

Comment:
`days * 40` looks like a new var, consider applying the approach to the rest of your values (the checklist, p.1)